daemon/src/projection.rs

```diff
@@ -87,7 +83,7 @@ struct State {
 }
 
 impl State {
-    pub fn to_cfd(&self) -> Vec<Cfd> {
+    pub fn as_cfds(&self) -> Vec<Cfd> {
```

Comment:
Nit: `as_` usually means "cheap conversion of references". `to_cfds` was correct (reference to owned conversion).

See https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv.

Reply:
thank you, I'll keep it in mind!